contracts/source/UniswapOracle.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity 0.6.3;
+
+import { BlockVerifier } from "./BlockVerifier.sol";
+import { MerklePatritiaVerifier } from "./MerklePatritiaVerifier.sol";
+import { Rlp } from "./Rlp.sol";
+import { IUniswapV2Pair } from "./IUniswapV2Pair.sol";
+import { UQ112x112 } from "./UQ112x112.sol";
+
+contract UniswapOracle {
+	using UQ112x112 for uint224;
+
+	uint256 private constant MIN_BLOCK_COUNT = 1;
```

Comment:
I'm assuming this is set to 1 for testing?

Reply:
If so, may be worthwhile to put a TODO comment on it to set it back, or making it not constant so we can set it in the constructor.

Reply:
I think making it set it constructor is right.

Reply:
I heard a rumor that Solidity vNext or latest has constructor constants...